tests/test_feedback.py

```
@@ -0,0 +1,521 @@
+            title, body = await format_feedback_with_ai(req, settings)
+        assert "Feature request:" in title
+        assert "Add export to CSV" in body
+
+    async def test_ai_returns_invalid_json_uses_fallback(self, settings):
+        req = FeedbackRequest(
+            feedback_type="bug",
+            description="Something broke",
+        )
+        with patch("jenkins_job_insight.feedback.call_ai_cli") as mock_ai:
+            mock_ai.return_value = AIResult(success=True, text="not json at all")
+            title, body = await format_feedback_with_ai(req, settings)
```

Ruff (0.15.12)

[warning] Unpacked variable `body` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)